libs/src/services/web3Manager/index.js

```diff
@@ -127,6 +127,19 @@ class Web3Manager {
     if (this.useExternalWeb3) {
       return ethSignTypedData(this.getWeb3(), this.getWalletAddress(), signatureData)
     } else {
+      // Due to changes in ethereumjs-util's toBuffer method as of v6.2.0
+      // non hex-prefixed string values are not permitted and need to be
+      // provided directly as a buffer.
+      // https://github.com/ethereumjs/ethereumjs-util/releases/tag/v6.2.0
+      Object.keys(signatureData.message).forEach(key => {
```

Comment:
nit: this guy really wants to be a `reduce` imo

Reply:
For each is cheaper since we wouldn't have to replace all the keys in a new object, unless you were thinking of mutating in the reduce at which point it's kinda the same though